AT/ad_thread/ad_routine.py

```diff
@@ -0,0 +1,138 @@
+import time
+from statsmodels.tsa import ar_model
+
+
+def compute_zone(value, info):
```

Comment:
The structure of this code could be improved by putting the zone = 2 in an else statement after all elifs